### PR TITLE
FIX change doc for release-2.5

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
Add known issue for 2.5.0 and add change log for 2.4.3

### Check List 

Tests 
 - No code

Related changes

 - Need to update the documentation

### CODE REVIEW

**Comment by shiyuhang0** [comment]
PTAL @xuanyu66 @zhangyangyu @sunxiaoguang

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ All notable changes to this project will be documented in this file.
 ### Documents
 - Fix in user docs forget to use options while using BatchWrite [#2169](https://github.com/pingcap/tispark/pull/2169)
 
+### Known issue
+- Spark-Sql's authentication and authorization through TiDB will affect other datasource such as hive.
```
Comment (xuanyu66) [inline CHANGELOG.md:18]: Please write an issue and link to it.  
Describe how this feature affect other datasources

File: docs/authorization_userguide.md [hunk 2]
```diff
@@ -60,5 +60,6 @@ Currently, Supported statements are as follows:
 
 ## limitations
 
+- Not supported with hive as datasource
```
Comment (xuanyu66) [inline docs/authorization_userguide.md:63]: The description is not accurate.
It's not only unsupported but also can't work with other data sources if you turn on auth mode

**Review by xuanyu66 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shiyuhang0) [reply CHANGELOG.md]: done

File: docs/authorization_userguide.md (hunk 2 above)
Comment (shiyuhang0) [reply docs/authorization_userguide.md]: done

File: docs/authorization_userguide.md [hunk 3]
```diff
@@ -60,5 +60,6 @@ Currently, Supported statements are as follows:
 
 ## limitations
 
+- Can't work with other datasource except tidb
```
Comment (zhangyangyu) [inline docs/authorization_userguide.md:63]: tidb -> TiDB

**Review by xuanyu66 — APPROVED** [review]

**Review by sunxiaoguang — APPROVED** [review]
LGTM

**Pull request merged**